components/form/FAutocompleteInput.vue

```diff
@@ -0,0 +1,373 @@
+<template lang="pug">
+FField.FAutocompleteInput(
+  :class="classes"
+  v-bind="{ name, label, labelTextColor, hint, hideHint, hintTextColor, hintIcon }"
+)
+  FMenu(
+    :options="matchingOptions"
+    :width="menuWidth"
+    :empty-text="loading ? loadingText : fieldValue ? noMatchText : emptyText"
+    :color="optionsMenuColor || color"
+    :text-color="optionTextColor"
+    :selected-option-color="selectedOptionColor"
+    :selected-option-text-color="selectedOptionTextColor"
+    :prevent-selection="preventSelection"
+    :disabled="disabled"
+    @select-option="handleSelectOption"
+    :loading="loading"
```

Comment:
It's best practice to put events at the bottom, after props
```suggestion
    :loading="loading"
    @select-option="handleSelectOption"
```